src/lib/hooks/use-query-params.ts

```diff
@@ -43,7 +43,7 @@ export const useQueryParams = <T extends Partial<UseQueryParamsArgs>>(
     const sell = searchParams.get('sell') || ''
 
     const queryNetwork =
-      chains.find((chain) => chain.id === network)?.id ?? chainId ?? 1
+      chains.find((chain) => chain.id === network)?.id ?? chainId
```

Comment:
It's safe to remove?

Reply:
Yes, the reason is that useNetwork handles this case, but we already sent 1 if there is no queryParam, which is wrong